stories/Atom/Lists/lists.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../assets/scss/variables";
+@import "../../assets/scss/breakpoints";
+
+/* unorderlist */
+ul {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}
+
+/* unorderlist */
+ol {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}
+
+/* description list */
+dl {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}

```

Comment:
If we have the same styling for all the tags then we can mention like ul, ol, dl